content/en/about/contributing/syntax/_index.md

```diff
@@ -22,6 +22,42 @@ For references The following shortcodes are available:
 - [Hugo's shortcodes](https://gohugo.io/content-management/shortcodes/)
 - [Docsy theme shortcodes](https://www.docsy.dev/docs/adding-content/shortcodes/)
 
+### Adding chapters
```

Comment:
I'm not sure how it is for other people, I'm a bit unsure about using the wording chapter here, to me it feels that's really making more sense for books than on the web. Section would sound better to me.

Reply:
OK, will update. For now I restored the current content.